app/controllers/bars_controller.rb

```diff
@@ -4,12 +4,20 @@ def index
     @bars = Bar.all
   end
 
+  def show
+    @bar = Bar.find(params[:id])
+    @review = Review.new
+    @reviews = @bar.reviews.order(created_at: :asc)
+  end
+
   def new
     @bar = Bar.new
   end
 
   def create
+    current_user
     @bar = Bar.new(bar_params)
+    @bar.user = @current_user
```

Comment:
Instead of adding `current_user` at the top, you can just make this line `@bar.user = current_user`. Fix in the next iteration